### PR TITLE
fix: remove unused validator stake view type

### PR DESCRIPTION
**Pull request by austinabell**
Unclear why this is here, opening it just to see if anything breaks, and see what does. Curious if it's a refactor that's broken this support, as I've gotten some server-side RPC errors relating to this type.

### CODE REVIEW

**Review by nikurt — APPROVED** [review]

**Comment by austinabell** [comment]
the context for this https://near.zulipchat.com/#narrow/stream/295558-pagoda.2Fcore/topic/Light.20client.20block.20queries.20failing/near/354053395

It's still unclear what this protocol change was, as I can't find any documentation on it.

**Comment by jakmeier** [comment]
@austinabell is this ready to merge? (you have the necessary approvals) If yes, do you see a merge button? Otherwise, I can add the auto merge label for you, I think we have to do it for external contributors.

**Comment by austinabell** [comment]
> @austinabell is this ready to merge? (you have the necessary approvals) If yes, do you see a merge button? Otherwise, I can add the auto merge label for you, I think we have to do it for external contributors.

I suppose so, and I can just open a nearcore issue for the failing queries below the ~55mil block range if so. I don't have a merge button

**Pull request merged**
